### PR TITLE
Feat/music

### PR DESCRIPTION
**Pull request by VincentLideboKjellman**

### CODE REVIEW

**Comment by SmokinClove** [comment]
Hi, somehow i am not getting sound now, even though i did before, could you help me take a look? Thanks a lot!

**Comment by VincentLideboKjellman** [comment]
I think it's because browsers don't allow sound to be played without permission. Although refreshing seems to ignore that

**Comment by SmokinClove** [comment]
Let me try again, because i am sure i could hear it previously
